### PR TITLE
ENG-2910: Add CreateGitSlicePullPlatform action

### PR DESCRIPTION
**Pull request by calinanca99**
This PR adds a new GitHub Action - _GitSlicePullPlatform_ - that is using the new GitSlice Service to execute the GitSlice Pull job. This is part of the release process for the GitSlice Service (see progress [here](https://linear.app/gitstart/issue/ENG-1864/release)).

### Changelog
- solve linting issues for the `GitSlicePush` and `GitSlicePull` jobs
- add `GitSlicePullPlatform` job

### CODE REVIEW

**Comment by calinanca99** [comment]
@ziahamza, could you please take a look at this? More specifically, at GitSlicePullPlatform/src/main.ts.

File: GitSlicePullPlatform/src/main.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    required: true
+  })
+
+  const is_open_source = core.getInput('is_open_source', {
+    required: false
+  })
+  const gitSliceFile = await fs.readFile('./git-slice.json')
+
+  // Prepare request
+  const endpoint = 'https://git-service.gitstart.dev/graphql'
```
Comment (ziahamza) [inline GitSlicePullPlatform/src/main.ts:47]: can this be conditional ... based on an env var perhaps? or perhaps we can go straight to .com?

Comment (calinanca99) [reply GitSlicePullPlatform/src/main.ts:47]: I'll change it to .com after I release to prod

Comment (jakewhelan) [inline GitSlicePullPlatform/src/main.ts:47]: Just curious, why not gateway?

Comment (calinanca99) [reply GitSlicePullPlatform/src/main.ts:47]: Good point :D

**Pull request merged**